src/clm/functions.py

```diff
@@ -231,7 +231,7 @@ def NeutraliseCharges(mol, reactions=None):
 
 def set_seed(seed):
     if seed is not None:
-        np.random.seed(seed)
+        random.seed(seed)
```

Comment:
Unless there's a reason for this, I'd still suggest using `np.random.seed(seed)` to avoid confusion with the built-in `random` module in python